FocusSublimeWindow.py

```diff
@@ -14,7 +14,6 @@ class FocusSublimeWindowCommand(WindowCommand):
     def run(self, **args):
         if NATIVE_FOCUS_WINDOW:
             self.window.bring_to_front()
```

Comment:
By the way I'd probably use feature detection rather than version sniffing. Something like:
```suggestion
        if hasattr(self.window, 'bring_to_front'):
            self.window.bring_to_front()
```

In JS I'd just write `self.window.bring_to_front?.()` but it doesn't look like Python has that

Reply:
Yeah, beacuse the (possible) environments in Web development are basically 
hostile and complex.  

Here in ST land, we would rather tell Package Control a ST3 version of 
this plugin, and then go on with ST4 without version sniffing, IMO.  Likely 
that ST3 version would be final too as the code is stable since years, well 
before I came at least.